reposerver/repository/repository_test.go

```diff
@@ -1194,8 +1194,8 @@ func TestGenerateHelmWithFileParameter(t *testing.T) {
 		ApplicationSource: &argoappv1.ApplicationSource{
 			Path: "./redis",
 			Helm: &argoappv1.ApplicationSourceHelm{
-				ValueFiles: []string{"values-production.yaml"},
-				Values:     `cluster: {slaveCount: 2}`,
+				ValueFiles:   []string{"values-production.yaml"},
+				ValuesObject: &runtime.RawExtension{Raw: []byte(`cluster: {slaveCount: 2}`)},
```

Comment:
Can we add a test case where both `Values` and `ValuesObject` both are being set to ensure that value of `Values` field is ignored?

Reply:
Done 😄